### PR TITLE
Actively thwart blank-slug saving...

### PR DESCRIPTION
**Pull request by mkoistinen**

### CODE REVIEW

**Comment by mkoistinen** [comment]
@jmelett @intellectronica Please review.

File: aldryn_newsblog/models.py [hunk 1]
```diff
@@ -220,6 +220,19 @@ def save(self, *args, **kwargs):
...
+        # article will fail Django validation on the form, since we are asking
+        # it to enforce uniqueness for (language, slug) pairs, and the slug in
+        # the form starts out empty. When Django tries to validate, it finds
+        # that the (language, slug) pair already exists, and we never even get
+        # here.
+        #
+        # Since the slug is derived from the title of the article, if the slug
+        # is still empty, then the title must be /effectively/ untitled.
+        if not self.slug:
+            self.slug = _('untitled-article')
```
Comment (intellectronica) [inline aldryn_newsblog/models.py]: What if this happens more than once? Will we then have uniqueness problems?

Comment (mkoistinen) [inline aldryn_newsblog/models.py]: The code further below will append things to make it unique.

**Comment by intellectronica** [comment]
:+1:

**Pull request merged**